### ISSUE TITLE
eu-west-2 is missing as a region

### ISSUE THREAD

**Issue by rsshilli** [aws/aws-sdk-java#945]
It looks like you announced a new London data center today but didn't add an eu-west-2 region to https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/java/com/amazonaws/regions/Regions.java

The error I'm receiving:
```
    java.lang.IllegalArgumentException: Cannot create enum from eu-west-2 value!
        at com.amazonaws.regions.Regions.fromName(Regions.java:77)
```

**Comment by kiiadi** [comment]
The Region enum update should be out in today's release. In the meantime can you use RegionUtils instead?

**Comment by rsshilli** [comment]
Not without rewriting our internal API :-(

**Comment by kiiadi** [comment]
We definitely recommend not using the enum for exactly this reason, we can't release new Regions to the enum before they're announced - which means there's a delay in getting the change out when new Regions are announced.

As I say the new enum should be out today so hopefully that should unblock you.

**Comment by hopel** [comment]
@kiiadi - not sure if this calls for a separate issue, hence asking here first: With release 1.11.67, eu-west-2 is available in the Region enum as you said, and also returned by e.g. `RegionUtils.getRegion("eu-west-2")`.

However, it is not contained in the list returned by `RegionUtils.getRegions()` or e.g. `RegionUtils.getRegionsForService("ec2")`. IIRC, this can be traced back to it not being listed in the [endpoints.json](https://github.com/aws/aws-sdk-java/blob/808c699c8c0aea7b6955a897524f4db51d75f4ac/aws-java-sdk-core/src/main/resources/com/amazonaws/partitions/endpoints.json) resource.

(For context/use case, we use `RegionUtils.getRegionsForService()` to populate region select widgets in our products UI, and right now eu-west-2 would not show up there, even though it is available.)

**Comment by kiiadi** [comment]
That will be in todays release - that particular piece of metadata missed our release cut-off yesterday and since it's not strictly necessary to call the services we roll-forward without it.

**Comment by kiiadi** [comment]
The enum and endpoints.json update is out with LHR now.
